src/core/Akka.Streams.TestKit/TestSubscriber_Fluent.cs

```diff
@@ -107,7 +107,6 @@ public ManualProbe<T> ExpectNext(TimeSpan? timeout, CancellationToken cancellati
             /// <summary>
             /// Fluent DSL. Expect multiple stream elements in arbitrary order.
             /// </summary>
-            [Obsolete("Use the method with CancellationToken support instead")]
```

Comment:
LGTM

Reply:
Not for the non-async stream.
For the async version, you will need to use the `AsyncBuilder()` helper
```
await subscriber.AsyncBuilder()
  .ExpectNextUnordered()
  .ExecuteAsync();
```